.gitignore

```diff
@@ -1,10 +1,10 @@
-.pytest_cache
-.repos
+\.pytest_cache
+\.repos
 venv
-.venv
-.idea
-.vscode
+\.venv
+\.idea
+\.vscode
```

Comment:
Why the `\`? Shouldn't it be `/`?

Reply:
Oh, sorry. My bad. I confused an expression with a regex and wanted to make sure the `.` is there :man_facepalming:.  Let's keep it as is.

Reply:
Yeah, glob vs regex always gets me as well. What's worse gitignore syntax is more than just glob :grimacing: